examples/saas-template/partner-integration/README.md

```diff
@@ -0,0 +1,389 @@
+> **🛈 NOTE:** You must use the same project you created previously.
+
+To create a new service account, follow the steps at [Creating and managing service accounts](https://cloud.google.com/iam/docs/creating-managing-service-accounts).
+
+
+#### Grant Billing Administrator permissions to your service account
+1. From the [Billing](https://console.cloud.google.com/billing) page, select the billing account you created previously.
+1. In the left navigation panel, click **Account management**.
+1. In the right pane, in the **INFO PANEL**, click **+ADD PRINCIPAL**.
+1. Set the new principals to be the service account and email address you created previously.
+1. Select the `Billing Account Administrator` IAM role
+
+
+#### Grant your service account access to Billing metering results
+1. Go to the [Service accounts](https://console.cloud.google.com/iam-admin/serviceaccounts) page.
+1. Select the project you created previously.
+1. Select the service account you created previously.
+1. Copy the **Email** field.
+1. Go to the [IAM](https://console.cloud.google.com/iam-admin/iam) page.
+1. Click **+ADD**.
+1. In the **New principals** text field, paste the service account email address you copied previously.
+1. Select the `BigQuery Admin` for the **role**.
+1. Click **SAVE**.
+
+
+#### Link your service account to call the Procurement API and report usage
+1. Go to the [Service accounts](https://console.cloud.google.com/iam-admin/serviceaccounts) page.
+1. Select the project you created previously.
+1. Select the service account you created previously.
+1. Copy the **Email** field.
+1. Go to the **Technical integration** section of your test product.
+1. Under **Link a service account to call Procurement API**, click **+ADD SERVICE ACCOUNT**.
+1. Paste the service account email that you copied previously.
+1. At the end of the input field, click **LINK**.
+1. Under **Add 'roles/servicemanagement.serviceController' to a service account**, click **+ADD SERVICE ACCOUNT**.
+1. Paste the service account email you copied previously.
+1. At the end of the input field, click **LINK**.
+
+
+### Setup BigQuery billing export
+> **🛈 NOTE:** You must use the same test billing account that you configured previously
+
+
+To set up billing export, follow the steps at [Set up Cloud Billing data export to BigQuery](https://cloud.google.com/billing/docs/how-to/export-data-bigquery#setup).
+
+
+---
+
+
+## Using `mpdev` to run the SaaS Test Tool
+
+
+### Before you begin
+To use `mpdev` to run the SaaS Test Tool, you must install both `Docker` and `mpdev`.
+
+1. To install Docker for Linux, follow the [Docker installation guide](https://docs.docker.com/engine/install/).
+1. To install mpdev, follow the installation instructions for [marketplace-tools](https://github.com/GoogleCloudPlatform/marketplace-tools#installation).
+1. To verify that docker can be found through your PATH environment variable, run the following command: `which docker`
```

Comment:
The `which` verify steps seem too obvious to be worth mentioning

Reply:
Thanks for the review. I've removed those steps.